Woodoku/src/Exceptions/Exceptions.py

```diff
@@ -0,0 +1,13 @@
+from mimetypes import init
```

Comment:
What is the use of this package?

Reply:
Hahaha, intellisense's leftover wreck 😅
Nice catch